pkgs/shells/bash/update-patch-set.sh

```diff
@@ -22,8 +22,10 @@ set -e
 
 rm -vf "$PATCH_LIST"
 
-wget "https://tiswww.case.edu/php/chet/gpgkey.asc"
-echo "4ef5051ce7200241e65d29c11eb57df8  gpgkey.asc" > gpgkey.asc.md5
+# https://savannah.gnu.org/projects/bash/: Group Admins: Chet Ramey
+# https://savannah.gnu.org/users/chet: Download GPG Key
+wget "https://savannah.gnu.org/people/viewgpg.php?user_id=2590" -O gpgkey.asc
+echo "d94dc6b8baa5b673971350236e95957f  gpgkey.asc" > gpgkey.asc.md5
```

Comment:
We should probably drop MD5 verification rather than updating it…

Reply:
Now updated to sha256